src/Core/Plugin.php

```diff
@@ -186,10 +186,15 @@ protected static function _loadConfig()
         if (Configure::check('plugins')) {
             return;
         }
-        try {
-            Configure::load('plugins');
-        } catch (\Exception $e) {
+
+        $vendorFile = dirname(dirname(dirname(dirname(__DIR__)))) . DS . 'cakephp-plugins.php';
```

Comment:
We had [a discussion](https://github.com/cakephp/cakephp/pull/5702#discussion-diff-23218074) about constant usage - and since this file is in the same path we need to find, it's just as easy to go up the dir-tree.